zebra-rpc/src/methods.rs

```diff
@@ -869,6 +989,21 @@ pub struct GetBlock(#[serde(with = "hex")] SerializedBlock);
 #[derive(Copy, Clone, Debug, Eq, PartialEq, serde::Deserialize, serde::Serialize)]
 pub struct GetBestBlockHash(#[serde(with = "hex")] block::Hash);
 
+/// Response to a `z_gettreestate` RPC request.
+///
+/// Contains the hex-encoded Sapling & Orchard note commitment trees, and their
+/// corresponding [`block::Hash`] and [`Height`].
+#[derive(serde::Serialize)]
+pub struct GetTreestate {
```

Comment:
@upbqdn unfortunately, we need to include `block.header.time` in the response, because `lightwalletd` expects it:
https://github.com/zcash/lightwalletd/blob/68789356fb1a75f62735a529b38389ef08ea7582/common/common.go#L108

(If you'd like to open a [zcashd](https://github.com/zcash/zcash) ticket to fix their docs, go for it!)

Reply:
This was a sneaky one! Thanks. I will open the ticket.